### PR TITLE
Fix arp proxy configuration for UDNs

### PR DESCRIPTION
**Pull request by jcaamano**

### CODE REVIEW

**Comment by jcaamano** [comment]
@qinqon PTAL

**Comment by qinqon** [comment]
@jcaamano I am not sure about this, it only make sense for L3 live migration, for L2 is not needed, I don't expect implementing live migration at UDN L3 anytime soon, but who knows...

**Comment by jcaamano** [comment]
@qinqon PTAL

File: dist/images/.gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@ ovn-kube-util
 ovnkube
 ovnkube-trace
 ovnkube-identity
+ovnkube-observ
```
Comment (qinqon) [inline dist/images/.gitignore:6]: This is not related.

**Review by qinqon — CHANGES_REQUESTED** [review]

File: dist/images/.gitignore (hunk 1 above)
Comment (jcaamano) [reply dist/images/.gitignore:6]: what do you want me to do?

Comment (qinqon) [reply dist/images/.gitignore:6]: I mean, I am ok adding those to this PR since the PR itself is super small, if you think is not worth it to do the .gitignore at different PR I am fine, we can go with it.

**Comment by qinqon** [comment]
> failing everywhere now
> 
> [FAIL] Services of type NodePort [AfterEach] should handle IP fragments

To extend on that, the error is 
```
[38;5;9m[FAILED] failed to remove ip address fc00:f853:ccd:e793:1111::0 from node ovn-control-plane, err: "failed to run \"docker exec ovn-control-plane ip addr delete fc00:f853:ccd:e793:1111::0/128 dev breth0\": exit status 2 (Error: ipv6: address not found.\n)"[0m
```

Don't get the relation, also those lanes are all about default network wich is not changed by this PR.

**Comment by qinqon** [comment]
Ahh looks like this is the fix:
- https://github.com/ovn-org/ovn-kubernetes/pull/4740

**Comment by jcaamano** [comment]
kubevirt is also failing now

**Review by trozet — APPROVED** [review]
